assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -131,7 +131,7 @@ resource "null_resource" "bootkube-start" {
   provisioner "remote-exec" {
     inline = [
       "sudo mv $HOME/assets /opt/bootkube",
-      "sudo systemctl start bootkube || (sudo journalctl -u bootkube && exit 1)",
+      "sudo systemctl start bootkube",
```

Comment:
The commit message has typos:

do -> does
losg -> logs

Reply:
:facepalm: fixed